scripts/shared/clusters.sh

```diff
@@ -4,6 +4,8 @@
 
 source ${SCRIPTS_DIR}/lib/shflags
 DEFINE_string 'k8s_version' '' 'Version of K8s to use'
+DEFINE_string 'olm_version' '0.14.1' 'Version of OLM to use'
```

Comment:
```suggestion
DEFINE_string 'olm_version' '0.15.0' 'Version of OLM to use'
```
FYI 0.15.0 was just released